skillbridge/src/component/SignUp/Login.tsx

```diff
@@ -1,7 +1,27 @@
-import '../SignUp/SignUp.css'
+import '../SignUp/SignUp.scss'
+import { useAuth } from "../../utils/AuthContext";
+import { useRef } from "react";
 import {SarahImg, forwardArrow, backwardArrow, googleLogo,loginArrow} from '../../assets/resource/imgResource'
+import { Link } from 'react-router-dom';
 
 function Login() {
+    const { user, loginUser } = useAuth()
+    //Add loginForm ref
+    const loginForm = useRef(null)
+
+    //Form submit handler
+    const handleSubmit = (e:any) => {
+        e.preventDefault()
+        const email = loginForm.current.email.value
+        const password = loginForm.current.password.value
+
+        const userInfo = { email, password }
+
+        loginUser(userInfo)
+
+        // const navigateHome = useNavigate();
+        // navigateHome('/home')
+    }
     return (
```

Comment:
If login is there in this folder, then folder names should be changed to Auth or Login should be moved to a different folder